core/trino-spi/src/test/java/io/trino/spi/TestSpiBackwardCompatibility.java

```diff
@@ -110,7 +111,9 @@ private static Set<String> getSpiEntities(ClassLoader classLoader, boolean inclu
             Class<?> clazz = classInfo.load();
             addClassEntities(entities, clazz, includeDeprecated);
         }
-        return entities.build();
+        return entities.build().stream()
+                .map(entity -> entity.replace(" final ", " "))
```

Comment:
Let's wisdom from @lukasz-stec 's  https://github.com/trinodb/trino/issues/12053#issuecomment-1104803210
and indicate it's a conscious choice

Reply:
Added a comment.